test/ManagementHistologyResults/cases/I44CIN1HistologyAfterHSILASCHCytology.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: I4.4 CIN1 Histology Test Result after HSIL or AscH Cytology
```

Comment:
Shouldn't this be a test for I.5 - CIN1 Diagnosed repeatedly for at least 2 years?

Reply:
I opened the following comment in the L2: https://mitre.sharepoint.com/:w:/r/sites/CervicalCancerCDS/Shared%20Documents/CDS%20Development/L2%20Artifacts/Working%20Documents/CDC_CC_CDS_L2Development_v7.docx?d=wc90d0228d64f420bb1565fb44a712630&csf=1&web=1&e=siGFR4&nav=eyJjIjo2ODc4NjQyNzl9. It appears that this recommendation and recommendation I.5 have some overlap, since both can deal with consecutive CIN1 results. However, I believe this one takes precedence, however need to confirm with @gmmeadows.